### PR TITLE
Move SELU to ATen

### PR DESCRIPTION
**Pull request by colesbury**
Fuse scale multiplication into ELU

### CODE REVIEW

File: torch/onnx/symbolic.py [hunk 1]
```diff
@@ -368,6 +368,10 @@ def elu(g, input, alpha, inplace=False):
     return g.op("Elu", input, alpha_f=_scalar(alpha))
 
 
+def selu(g, input, inplace=False):
```
Comment (ezyang) [inline torch/onnx/symbolic.py]: No inplace arg here.

**Comment by ezyang** [comment]
CI failure looks legit.

File: aten/src/THCUNN/ELU.cu [hunk 2]
```diff
@@ -6,61 +6,52 @@
...
-    : alpha_(alpha)
+  ELUupdateOutput_functor(T poscoef, T negcoef)
+    : poscoef_(poscoef)
+    , negcoef_(negcoef)
   {}
 
   __device__ void operator()(T *output, const T *input) const
   {
-    *output = *input <= 0 ? (exp(*input) - 1) * alpha_ : *input;
+    *output = *input <= 0 ? (exp(*input) - 1) * poscoef_ : *input * negcoef_;
```
Comment (gchanan) [inline aten/src/THCUNN/ELU.cu]: what's the  definition of poscoef?  You multiply the negative inputs by the poscoef?

Comment (colesbury) [reply aten/src/THCUNN/ELU.cu]: oops... messed up my positives and negatives

**Review by gchanan — APPROVED** [review]

**Pull request merged**